jerry-core/jerryscript.h

```diff
@@ -389,6 +389,11 @@ size_t jerry_parse_and_save_literals (const jerry_char_t *source_p, size_t sourc
                                       uint32_t *buffer_p, size_t buffer_size, bool is_c_format);
 
 /**
+ * External context functions.
+ */
+size_t jerry_get_context_size (void);
+size_t jerry_get_lcache_size (size_t row_count, size_t row_length);
```

Comment:
My proposal:
* put the `table` of `jerry_hash_table_t` into `jerry_context_t`, and remove the jerry_global_hash_table. In other words, lcache become the member of jerry_context

* since the `ECMA_LCACHE_HASH_ROWS_COUNT` and `ECMA_LCACHE_HASH_ROW_LENGTH` is defined in `ecma-globals.h`, and cannot be changed via public api or config option. So in this patch, we still don't expose the size/existence of the lcache to public.